etl/steps/data/garden/forests/2024-05-08/ifl.py

```diff
@@ -0,0 +1,46 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+REGIONS = ["North America", "South America", "Europe", "Africa", "Asia", "Oceania"]
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("ifl")
+    # Load regions dataset.
+    ds_regions = paths.load_dataset("regions")
+    # Read table from meadow dataset.
+    tb = ds_meadow["ifl"].reset_index()
+
+    #
+    # Process data.
+    #
+    # Convert to square kilometers.
```

Comment:
Maybe, clarify in the comment that the original data is in 1000 square kilometers.